### PR TITLE
Fix SSR: Only access localStorage if window is defined

### PR DESCRIPTION
**Pull request by danlutz**
In server-side-rendered applications, window is not defined during build time, because they are executed in a Node environment. This PR adds checks for window before calling any localStorage function.

I also ran `npm run babelify`, because I saw that the browser folder also is commited to the source control, hope that is ok 👻

PS: Thanks for providing this great service! Keep going 💪🏼

closes https://github.com/sw-yx/react-netlify-identity/issues/4

### CODE REVIEW

File: browser/gotrue.js [hunk 1]
```diff
@@ -1 +1 @@
...
...,{key:"_details",get:function get(){var userCopy={};for(var key in this){if(key in User.prototype||key in forbiddenSaveAttributes){continue}userCopy[key]=this[key]}return userCopy}}],[{key:"removeSavedSession",value:function removeSavedSession(){isBrowser()&&localStorage.removeItem(storageKey)}},{key:"recoverSession",value:function recoverSession(apiInstance){if(currentUser){return currentUser}var json=isBrowser()&&localStorage.getItem(storageKey);if(json){try{var data=JSON.parse(json);var url=data.url,token=data.token,audience=data.audience;if(!url||!token){return null}var api=apiInstance||new _microApiClient2.default(url,{});return new User(api,token,audience)._saveUserData(data,true)}catch(ex){console.error(new Error("Gotrue-js: Error recovering session: "+ex));return null}}return null}}]);return User}();exports.default=User;function urlBase64Decode(str){var output=str.replace(/-/g,"+").replace(/_/g,"/");switch(output.length%4){case 0:break;case 2:output+="==";break;case 3:output+="=";break;default:throw"Illegal base64url string!"}var result=window.atob(output);try{return decodeURIComponent(escape(result))}catch(err){return result}}},{"./admin":3,"micro-api-client":1}]},{},[4]);
```
Comment (sw-yx) [inline browser/gotrue.js]: hmm.. were you forced to modify this file? surprised to see that we checked in this build output here. but i dont know this library that well.

**Comment by sw-yx** [comment]
cc @bcomnes this is helpful for gotruejs to be used in SSR/SSG environments.

File: browser/gotrue.js (hunk 1 above)
Comment (danlutz) [reply browser/gotrue.js]: No I was not forced, but I saw it was commited, so I compiled and pushed it too 👍

**Comment by sw-yx** [comment]
thank you dan!

**Comment by sw-yx** [comment]
@danlutz published as v0.9.25

**Pull request merged**